### ISSUE TITLE
FMS VDA5050 packet reciever

### ISSUE THREAD

**Issue by JonasPetersenFAU** [amosproj/amos2022ss03-turtlebot-fleet-management#60]
## User story
1. As a SD
2. I want / need to build a packet reciever
3. So that incoming MQTT VDA5050 packets (JSON) should be converted to a Python object (defined in vda5050.py)

## Acceptance criteria
* Shall be done for the "state" and "connection" packets
* Packets will be recieved and converted

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
